docs/source/tutorial/kernel.rst

```diff
@@ -370,3 +372,69 @@ You can specify the kernel name by using the ``kernel_name`` keyword argument as
 
 .. note::
    Currently, :func:`cupy.fuse` can fuse only simple elementwise and reduction operations.  Most other routines (e.g. :func:`cupy.matmul`, :func:`cupy.reshape`) are not supported.
+
+.. _jit_kernel_definition:
+
+JIT kernel definition
+---------------------
+
+The :class:`~cupyx.jit.rawkernel` decorator can define raw CUDA kernel from Python function.
+
+In this section, the Python function wrapped with the decorator is called as *target function*.
+
+The target function consists of the elemental scalar operations, and the users have to manage how parallelize them. CuPy's array operations which automatically parallelize the operation (e.g., :func:`cupy.add`, :func:`cupy.sum`) is not supported. If the custom kernel from the array functions are desired, please refer the :ref:`kernel_fusion` section.
+
+Basic Usage
+^^^^^^^^^^^
+
+:class:`cupyx.jit.rawkernel` decorator can define a CUDA global function.
```

Comment:
I feel this line is repetitive of what's been said in the beginning of the section. How about a more descriptive sentence to introduce the example?
```suggestion
Here is a short example for how to write a :class:`cupyx.jit.rawkernel` to copy the values from ``x`` to ``y`` using a grid-stride loop:
```